2013-01-15 22:55:26.148614: ENGINE starting
2013-01-15 22:55:26.151558: ENGINE loading STUDENT ENGINE part 2
2013-01-15 22:55:26.166830: ENGINE loaded player module: OKComputer as player 1
2013-01-15 22:55:26.167095: QUORIDOR ENGINE CLIENT 1.0.5 by Adam Oest initializing
2013-01-15 22:55:26.665616: ENGINE calling STUDENT ENGINE init
2013-01-15 22:55:26.666367: STUDENT ENGINE init call successfully returned
2013-01-15 22:55:26.666699: ENGINE calling init() for player 1
2013-01-15 22:55:26.667103: OKComputer initialized as player 1
2013-01-15 22:55:26.681813: ENGINE GUI accessible at:
https://adelaide.cs.rit.edu/~royale/quoridorui/gameview/gameid.45440/partone.0/apikey.36bc8c65d1d4af66485/public.c0b92a12f95edc6b27c7b11db58243747351553950f628ce61bfe/animations.500/
2013-01-15 22:55:26.682405: ENGINE processing move # 1
2013-01-15 22:55:26.683166: ENGINE getting move for player 1 from PRE_MOVE list 
2013-01-15 22:55:26.683539: ENGINE validating player 1 move. Player 1 Wall: [7,6] -> [9,6]
2013-01-15 22:55:26.683755: ENGINE calling STUDENT ENGINE validate_move
2013-01-15 22:55:26.930785: ERROR: list index out of range
2013-01-15 22:55:26.955366: TRACE: 
Traceback (most recent call last):
  File "C:\Users\Adam\Desktop\a\Quoridor_Client_2121\Releases\1.0.5\Python3.2\Model\__init__.py", line 560, in studentEngineCall
  File "/home/wheeler/workspace/quoridor_group/StudentEngines/jak3122/__init__.py", line 374, in validate_move
    player_goals[0][i][0],\
IndexError: list index out of range

2013-01-15 22:55:26.956632: ERROR: 2013-01-15 22:55:26.956596: ENGINE exiting due to STUDENT ENGINE exception

2013-01-15 22:55:26.957393: TRACE: 
Traceback (most recent call last):
  File "C:\Users\Adam\Desktop\a\Quoridor_Client_2121\Releases\1.0.5\Python3.2\Engine\controller.py", line 246, in nextTurn
  File "C:\Users\Adam\Desktop\a\Quoridor_Client_2121\Releases\1.0.5\Python3.2\Model\__init__.py", line 567, in studentEngineCall
Engine.security.GameException: 2013-01-15 22:55:26.956596: ENGINE exiting due to STUDENT ENGINE exception


2013-01-15 22:55:27.830482: 2013-01-15 22:55:26.956596: ENGINE exiting due to STUDENT ENGINE exception

2013-01-15 22:55:27.832211: ENGINE finished with next move
2013-01-15 22:55:27.833801: ENGINE shutting down
